example_test.go

```diff
@@ -25,15 +23,15 @@ func ExampleConn_Bind() {
 
 // ExampleConn_Search demonstrates how to use the search interface
 func ExampleConn_Search() {
-	l, err := ldap.Dial("tcp", fmt.Sprintf("%s:%d", "ldap.example.com", 389))
```

Comment:
find/replace victim: `"ldap.example.com"`

Reply:
You are completely right, I will fix this immediately.